core/src/main/java/org/apache/accumulo/core/client/impl/Tables.java

```diff
@@ -149,6 +149,28 @@ public static void clearCache(Instance instance) {
     getZooCache(instance).clear(ZooUtil.getRoot(instance) + Constants.ZNAMESPACES);
   }
 
+  /**
+   * Clears the zoo cache from instance/root/{PATH}
+   *
+   * @param instance
+   *          The Accumulo Instance
+   * @param zooPath
+   *          A zookeeper path
+   */
+  public static void clearCacheByPath(Instance instance, final String zooPath) {
+
+    String thePath;
+
+    if (zooPath.startsWith("/")) {
+      thePath = zooPath;
+    } else {
+      thePath = "/" + zooPath;
+    }
+
+    getZooCache(instance).clear(ZooUtil.getRoot(instance) + thePath);
```

Comment:
Do you know if this zoocache method handles path normalization?  For example will is treat `/a/b/c/` and `/a//b/c/` the same?